portfolio/src/main/webapp/script.js

```diff
@@ -95,19 +96,28 @@ function renderForm() {
       authDiv.innerHTML = "<p>Login <a href=\"" + res.url + "\">here</a> to leave comments.</p>"
     }
   });
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
+      commentForm.lat.value = position.coords.latitude;
+      commentForm.lng.value = position.coords.longitude;
+    });
+  }
 }
 
-function initMap() {
-  const map = new google.maps.Map(document.getElementById('map'), {
+var map; 
```

Comment:
nit: `const` when you don't need to reassign the value (which you should generally strive for) or `let` when you do: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
fixed! Would like to ask more about var vs let tomorrow.